release-please-config.json

```diff
@@ -384,6 +384,10 @@
       "component": "google-cloud-profiler",
       "skip-github-release": true
     },
+    "java-publicca": {
+      "component": "google-cloud-publicca",
+      "skip-github-release": true
```

Comment:
What does `skip-github-release` do?

Reply:
Not to create a GitHub release entry for now during the shadow mode.